test/kubernetes_test.go

```diff
@@ -1,9 +1,8 @@
-// +build k8s
```

Comment:
Dont remove this line, its a build constraint ... see https://golang.org/pkg/go/build/
It is used by the make file to control when this test is executed.